docker/cron/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.fedoraproject.org/fedora:38
+MAINTAINER copr-devel@lists.fedorahosted.org
+
+RUN dnf -y update
+RUN dnf -y install cronie tar
```

Comment:
Every command create new layer in container. Especially dnf command populates /var/cache/dnf and create new layer with zillion files in cache. It is recommended to run it like:
RUN dnf -y update && dnf -y install cronie tar && dnf clean all